frontend/src/pages/Admin/Categories.js

```diff
@@ -0,0 +1,23 @@
+import { Typography } from "@mui/material";
+
+import { useAuth } from "../../hooks/auth";
+import AdminLayout from "./AdminLayout/AdminLayout";
+import PagePreloader from "../../components/PagePreloader";
+
+const Categories = () => {
+  const { user } = useAuth();
+
+  return (
+    <AdminLayout navTitle="Manage Categories">
+      {!user ? (
+        <PagePreloader loading={!user} size={30} />
```

Comment:
`<PagePreloader loading={!user} size={30} />`
This is quite repetitive, why not include this in the AdminLayout? And pass identifier if this should be displayed or the actual content? Update rest of pages

Reply:
Ahh, I did that on purpose Ma'am , but I will be removing the `<PagePreloader loading={!user} size={30} />` in the other pages such as the Categories.js, Questions.js and Users.js in the current task I'm doing right now.

Reply:
Ah if there is another plan for implementation, then put a `TODO` comment, with enough context, so that reviewer/team mates can understand goal of the code.